scripts/poweron/IPMI.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
This appears to be very simple code, could it be a shell-script instead (with proper quoting)? Would be less maintenance than Python (although I hope migrations between minor python versions will be less disruptive than 2 to 3).

Reply:
So the code here is essentially the original DRAC.py modified to use ipmitool rather than the DRAC utility, I didn't write it fresh for this.

Reply:
(It gets invoked via power-on.py as a python method call (rather than as an actual script), as power-on.py then waits for the host to come live before completing the plugin call)

Reply:
How about the comment from the CI? Are they worth addressing to quiet them?